src/volumes/service.go

```diff
@@ -14,6 +14,7 @@ var (
 	ErrAlreadyAttached     = errors.New("volume is already attached")
 	ErrNotAttached         = errors.New("volume is not attached")
 	ErrAttachLimitReached  = errors.New("max number of attachments per server reached")
+	ErrLockedServer        = errors.New("server is locked")
```

Comment:
I think this part can be included in a PR about fixing the locked server errors CSI-driver receives from the hcloud.